src/components/PromotionCard.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { RiTShirtLine, RiMotorbikeLine } from 'react-icons/ri';
+import { GiPerfumeBottle } from 'react-icons/gi';
+import { IconContext } from 'react-icons';
+
+export const PromotionCard = (props) => {
+    const { promotionsListData } = props;
+    // console.log("PromotionCard", promotionsListData);
+
+    if (promotionsListData.name === "beauty")
+        return (
+            <BottomImagePromotion
+                promotionsListData={promotionsListData}
+            />
+        )
+
+    else
+        return (<TopImagePromotion
+            promotionsListData={promotionsListData}
+        />)
+};
+
+const TopImagePromotion = (props) => {
+    const { promotionsListData } = props;
+    // console.log("TopImagePromotion", promotionsListData);
+    return (
+        <>
+            {promotionsListData.map((item) =>
+                item.name === "beauty" ?
```

Comment:
- This condition will not be true in any case as we checked the opposite of it before rendering this component (else..).